modules/UI/util/MessageHandler.js

```diff
@@ -453,28 +455,19 @@ const messageHandler = {
     },
 
     /**
-     * Shows a dialog prompting the user to send an error report.
-     *
-     * @param titleKey the title of the message
-     * @param msgKey the text of the message
-     * @param error the error that is being reported
+     *  Shows an error dialog to the user.
+     * @param {string} titleKey - The title of the message.
+     * @param {string} msgKey - The text of the message.
```

Comment:
I think the docs are incorrect because it takes in props and not two strings. I probably screwed this one up initially.